addons/payment_mollie/models/payment_acquirer.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+import logging
+
+import requests
+from werkzeug import urls
+
+from odoo import _, api, fields, models, service
+from odoo.exceptions import ValidationError
+
+from odoo.addons.payment_mollie.const import MOLLIE_CURRENCIES
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    provider = fields.Selection(
+        selection_add=[('mollie', 'Mollie')], ondelete={'mollie': 'set default'}
+    )
+    mollie_api_key = fields.Char(
+        string="API key",
+        help="The Test or Live API key depending on the configuration of the acquirer",
+        required_if_provider="mollie", groups="base.group_user"
+    )
```

Comment:
I use `API key` because there is one field in the `payment_adyen` module which is also having the same label and it's giving this warning 

> Two fields (adyen_api_key, mollie_api_key) of payment.acquirer() have the same label: API Key

Reply:
Indeed, then please change the `string` in the field definition to `Mollie API Key` and to `API Key` in the view.

Reply:
Done :+1: